libraries/bxdf/gltf_pbr.mtlx

```diff
@@ -104,9 +106,33 @@
 
     <!-- Roughness -->
 
-    <roughness_anisotropy name="roughness_uv" type="vector2">
-      <input name="roughness" type="float" interfacename="roughness" />
-    </roughness_anisotropy>
+    <combine2 name="roughness_uv" type="vector2" nodedef="ND_combine2_vector2">
```

Comment:
By convention, we omit these `nodedef` attributes in our graph definitions, as they can be inferred from the signature of the node.  That makes our standard libraries more robust to future changes in nodedef names, though in practice it's very rare that such a change would occur.